### ISSUE TITLE
Run as service or daemon

### ISSUE THREAD

**Issue by poliveira89** [docker/swarm#1638]
After successfully install it by manual setup. I made the typical tests: `swarm version` and ...
```bash
swarm join --advertise=192.168.100.10:2375 token://<token>
INFO[0000] Registering on the discovery service every 1m0s...  addr=192.168.100.10:2375 discovery=token://<token>
INFO[0060] Registering on the discovery service every 1m0s...  addr=192.168.100.10:2375 discovery=token://<token>
INFO[0121] Registering on the discovery service every 1m0s...  addr=192.168.100.10:2375 discovery=token://<token>
...
```

But the problem is that stays attached to the terminal/tty. I was wondering (and submitted this after searching for it), if I'm able to run any subcommand of Swarm as daemon/service similar to `-d` from `docker run`

EDIT: I forgot to mention, I can run this like this:
```
swarm join --advertise=192.168.100.10:2375 token://<token>&
```
or
```
swarm join --advertise=192.168.100.10:2375 token://<token> > /dev/null 2>&1
```
But I was wondering if it's a way more elegant.

**Comment by dongluochen** [comment]
@poliveira89 Thanks for the suggestion. We will take this into consideration in planning.

**Comment by poliveira89** [comment]
@dongluochen it was a question but I'm glad you found it interesting or at least feasible for consideration.  :smiley:

**Comment by abronan** [comment]
Related to #1568

**Comment by mstanleyjones** [comment]
I think this is fixed by #1568. If this is wrong, please reopen.
